### PR TITLE
[#483] Use FormConsole and FormFailedDialog for login

### LINKED ISSUE #483: Use FormConsole and FormFailedDialog for login

### PR DESCRIPTION
**Pull request by sharon-ho**
# General info

**Issue number**: closes #483 

**Task description**: 
- Refactor login to use FormConsole and FormFailedDialog

# Testing

**Test coverage**: 100%

# Additional notes

**Note to reviewers**:
N/A

**To do before merging**:
N/A

### CODE REVIEW

File: frontend/src/login/__tests__/LoginFormConsole.test.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    describe("the <FormConsole/> handleSubmit", () => {
+      it("to be equal to the <CreateUserFormConsole/>'s handleSubmit prop", () => {
+        const loginFormConsole = wrapper.find(FormConsole);
+
+        expect(loginFormConsole.props().handleSubmit).toBe(
+          wrapper.props().handleSubmit
+        );
+      });
+    });
```
Comment (felixlapierre) [inline frontend/src/login/__tests__/LoginFormConsole.test.jsx]: Describes are kinda weird here
"the FormConsole handleSubmit to be equal to the CreateUserFormConsole's handleSubmit prop"
Can we just stick to the usual "one FormConsole with the right props" that we usually do?

**Review by felixlapierre — DISMISSED** [review]
one comment that digs underground for rare minerals otherwise LGTM

File: frontend/src/login/__tests__/LoginFormConsole.test.jsx (hunk 1 above)
Comment (sharon-ho) [reply frontend/src/login/__tests__/LoginFormConsole.test.jsx]: 6c075c2

**Review by gdzooks — DISMISSED** [review]
LGTM!

**Review by gdzooks — APPROVED** [review]

**Review by felixlapierre — APPROVED** [review]

**Pull request merged**